File: Tarea7U2/src/main/java/com/mycompany/tarea7u2/Bank.java

package com.mycompany.tarea7u2;

public class Bank {
    String accounts [] = new String [15];
    long accounts1 [] = new long [15];
    int z = 0, j = 1;
    private long accountNumber;
    private char tipe;
    private String c;
    boolean band = false;

    public Bank() {
    }

    public Bank(long accountNumber, char tipe) {
        this.accountNumber = accountNumber;
        this.tipe = tipe;
    }
    
    public void Array(long accountNumber, char tipe, String name){
        String x = String.format(j + ". Account number: " + accountNumber + "\nTipe of account: " + tipe + "\nOwner: " + name+ "\n");
        accounts[z] = x;
        accounts1[z] = accountNumber;
        z++;
        j++;
        System.out.println("Saved");
    }
    
    public void showAccounts(){
        int h = 0;
        for (; h < z; h++){
       System.out.println(accounts[h]);
           
        }
    }
    
    public void consultAccounts(long accountNumber1){
        int d = 0;
        for (; d < z  && band == false ;){
            if (accounts1[d] == accountNumber1){
                c = accounts[d];
                band = true;
            }
            else{
                d++;
            } 
        }
        
        if(band == true){     
            System.out.println(c);
            band = false;
        }
        else{
            System.out.println("\nAccount don't exist.");
        } 
    }
 }
    

File: Tarea7U2Python/Bank.py
class Bank:
    
    def __init__(self):
        self.accounts = []

    def add_account(self, account_number, account_type, owner):
        account_info = f"Account number: {account_number}\nAccount type: {account_type}\nOwner: {owner}\n"
        self.accounts.append(account_info)

    def show_accounts(self):
        if self.accounts:
            print("--- Accounts ---")
            for account in self.accounts:
                print(account)
        else:
            print("No accounts found.")

    def consult_account(self, account_number):
        for account in self.accounts:
            if account_number in account:
                print("Account information:")
                print(account)
                return
        print("Account not found.")